libs/txnbuild/whitelist.go

```diff
@@ -49,6 +52,8 @@ func (whitelist *Whitelist) FromXDR(vXdrOp vxdr.VeloOp) error {
 	return nil
 }
 
+// Validate for Whitelist validates the required struct fields. It returns an error if any of the fields are
```

Comment:
Validation function for Whitelist. Validates the required struct fields. It returns an error if any of the fields are